src/core/transaction.zig

```diff
@@ -227,18 +225,19 @@ pub const Message = struct {
     pub const @"!bincode-config:account_keys" = shortVecConfig([]const Pubkey);
     pub const @"!bincode-config:instructions" = shortVecConfig([]const CompiledInstruction);
 
-    pub fn default() Message {
-        return Message{
-            .header = MessageHeader{
-                .num_required_signatures = 0,
-                .num_readonly_signed_accounts = 0,
-                .num_readonly_unsigned_accounts = 0,
-            },
-            .account_keys = &[_]Pubkey{},
-            .recent_blockhash = Hash.generateSha256Hash(&[_]u8{0}),
-            .instructions = &[_]CompiledInstruction{},
-        };
-    }
+    pub const empty: Message = .{
```

Comment:
fine with constant but would prefer to be called default - same reasoning as other comment

Reply:
Same reply as other comment, let's maintain the discussion under that one.